service/service.go

```diff
@@ -0,0 +1,16 @@
+package service
+
+import (
+	"github.com/klaytn/klaytn/networks/p2p"
+)
+
+type Service interface {
+	// Start starts the service.
+	Start() error
+
+	// Stop stops the service.
+	Stop() error
+
+	// HandleMsg handles the message.
+	HandleMsg(p2p.Msg, chan error)
```

Comment:
Other services don't need the HandleMsg function.

Reply:
@kjeom I thought that it may be necessary for other services to process txMsg, or blockMsg including headers and bodies. We may change them if not necessary.